src/readqps.jl

```diff
@@ -700,6 +700,11 @@ function read_quadobj_line!(qps::QPSData, card::MPSCard)
 end
 
 function readqps(filename::String; mpsformat::Symbol=:free)
+    qps = open(filename, "r")
```

Comment:
Shouldn't there be a `close` at some point?

Reply:
Good point. The IO version of `readqps` was closing the stream, but that doesn't seem right. Rearranged this as an open(...) do block.